pages/desktop/consumer_pages/details.py

```diff
@@ -40,15 +40,15 @@ class Details(Base):
     _report_abuse_button_locator = (By.CSS_SELECTOR, '.button.abuse')
     _report_abuse_box_locator = (By.CSS_SELECTOR, '.abuse-form')
 
-    def __init__(self, testsetup, app_name=False, first_access=False):
+    def __init__(self, testsetup, app_name=None):
         Base.__init__(self, testsetup)
-        if first_access:
```

Comment:
I assume from these changes that this `first_access` thing is just totally unnecessary. The default was `False` so that would mean that the __init__ would, by default, not call `self.wait_for_page_to_load()` whereas now it is always calling `self.wait_for_page_to_load()`. Do you have any idea why this was added in the first place?